src/develop.js

```diff
@@ -39,7 +41,7 @@ const List = () => {
     trashIcon.className = 'fas fa-trash-alt icon2';
     list.append(checkboxes, listText, threeDots, trashIcon);
     // Add event to checkboxes
```

Comment:
- [ ] Kindly as suggested by the previous reviewer make sure the completed todos persist in refreshing. this means when a todo is marked as completed and the page is refreshed, the todo should stay marked as completed. 

https://github.com/Jessemccallums/set-up-webpack/pull/4#discussion_r917203404

Reply:
Sir, I already checked it and it's working as the requirements needs

Reply:
Hello @Jessemccallums

Reply:
Hello @andersonlebon , I checked my project and everything is working well with the requeriments, what do you suggest me to do?

Reply:
[Screencast from 07-09-2022 06:45:27 AM.webm](https://user-images.githubusercontent.com/65068771/178092025-a14b7a21-c767-42b4-bf26-bcf6a3916ec9.webm)